grails-app/domain/org/pih/warehouse/order/OrderAdjustment.groovy

```diff
@@ -104,6 +104,20 @@ class OrderAdjustment implements Serializable, Comparable<OrderAdjustment> {
         return !postedPurchaseInvoiceItems.empty
     }
 
+    /**
+     * Overall invoiced quantity for this adjustment. Expected is either 0 or 1. Should not be more than 1.
+     * */
+    Integer getInvoicedQuantity() {
+        return invoiceItems?.findAll { it.invoice.isRegularInvoice && !it.inverse }?.sum { it.quantity } ?: 0
+    }
+
+    /**
+     * Overall inversed quantity for this adjustment. Expected is either 0 or 1. Should not be more than 1.
```

Comment:
with that being said, I'm wondering if we shouldn't throw any exception if it turns out to be more than 1?

Reply:
sounds good